submissions/nwokejeme-immaculeta/database/init.sql

```diff
@@ -0,0 +1,127 @@
+// CREATE DATABASE
+	CREATE DATABASE imma_classique;
+	USE imma_classique;
+    
+// CREATE ADMINS TABLE
+CREATE TABLE admins (
+    admin_id INT NOT NULL AUTO_INCREMENT,
+	name VARCHAR(100) NOT NULL,
+	email VARCHAR(50) NOT NULL,
+	password VARCHAR(20) NOT NULL,
+	phone_number VARCHAR(15) NOT NULL,
+	PRIMARY KEY (admin_id)
+);
+
+// INSERT RECORD INTO ADMINS TABLE
+INSERT INTO admins (name, email, password, phone_number,)
+VALUES  ('Hannah Donald', 'hannahdonald@gmail', 'july31', '+234808437228'),
+	('Michael Jackson', 'mikejacson@yahoo.com', 'micson3', '+23490893144879'),
+	('Donald Trump ', 'donaldtrump@sample.com', 'trump4life', '+2344903002'),	
+
+// CREATE CATEGORIES TABLE
+CREATE TABLE categories (
+    category_id INT NOT NULL AUTO_INCREMENT,
+	category_name VARCHAR(50) NOT NULL,
+	PRIMARY KEY (category_id)
+);
+
+// INSERT DATA INTO CATEGORIES TABLE
+INSERT INTO categories (category_name)
+VALUES 	('Bag')
+	('Belt'),
+	('Wristwatch'),
+    ('sunglasses'),
+					                   
+// CREATE PRODUCTS TABLE
+CREATE TABLE products (
+    id INT NOT NULL AUTO_INCREMENT,
```

Comment:
be consistent, use `product_id` rather than just `id` since that's how you want to write your primary key column name (i.e tablename_id)